src/server/accesscontrol.ts

```diff
@@ -0,0 +1,28 @@
+import {AccessControl} from 'accesscontrol'
+
+export const accessControl = new AccessControl()
+
+export const allRoles = [
+  'user',
+  'pro',
+  'instructor',
+  'editor',
+  'admin',
+  'publisher',
+  'editor',
+  'instructor',
+  'reviewer',
+  'advocate',
+]
+
+accessControl.grant('user')
+accessControl.grant('pro').extend('user')
+accessControl.grant(['editor', 'reviewer']).extend('pro')
+accessControl.grant(['advocate']).extend('reviewer')
+accessControl.grant(['instructor']).extend('pro')
+accessControl.grant('publisher').extend(['editor', 'instructor'])
+accessControl.grant('admin').extend(['editor', 'instructor', 'publisher'])
```

Comment:
kind of goofy, but they have other better ways, just wanted to get it in for now

Reply:
🚮